launch/client.py

```diff
@@ -740,6 +744,7 @@ def create_model_endpoint(
                 max_workers=max_workers,
                 per_worker=per_worker,
                 gpu_type=gpu_type,
+                high_priority=high_priority,
```

Comment:
In this case, I think you'll want to change `CreateModelEndpointRequest` to `CreateModelEndpointRequestV1` in line 805. I think you'll get an error if you try to create a `CreateModelEndpointRequest` as is